tests/test_checkout.py

```diff
@@ -32,12 +32,19 @@ def tearDownClass(cls):
     def setUp(self):
         dss.Config.set_config(dss.BucketConfig.TEST)
         self.s3_test_fixtures_bucket = get_env("DSS_S3_BUCKET_TEST_FIXTURES")
-        self.s3_test_checkout_bucket = get_env("DSS_S3_CHECKOUT_BUCKET_TEST")
-        self.s3_test_bucket = get_env("DSS_S3_BUCKET_TEST")
+        self.gs_test_fixtures_bucket = get_env("DSS_GS_BUCKET_TEST_FIXTURES")
+
+    def get_test_fixture_bucket(self, replica: Replica) -> str:
+        if replica == Replica.aws:
+            bucket = self.s3_test_fixtures_bucket
+        elif replica == Replica.gcp:
+            bucket = self.gs_test_fixtures_bucket
+        return bucket
 
     @testmode.integration
     def test_sanity_check_valid(self):
-        self.launch_checkout(self.s3_test_checkout_bucket)
+        for replica in Replica:
+            self.launch_checkout(replica.checkout_bucket, replica)
 
     @testmode.integration
     def test_pre_execution_check_doesnt_exist(self):
```

Comment:
does this test need to be made x-replica?

Reply:
this is an obsolete test - deleted